src/components/__tests__/__snapshots__/AppNavigation.test.js.snap

```diff
@@ -78,7 +82,9 @@ exports[`AppNavigation.vue renders the correct markup 1`] = `
           <v-icon-stub
             size="20"
           >
-            mdi-settings
+            
```

Comment:
why are there new lines?

Reply:
I think something changed in the Vuetify library. Looks like they added breaking spaces in the default slot of the `VIcon` component.